man/systemd.network.xml

```diff
@@ -734,6 +734,16 @@
           Defaults to false.
           </para>
         </listitem>
+      </varlistentry>
+       <varlistentry>
+        <term><varname>IgnoreCarrier=</varname></term>
```

Comment:
hmm, isn't "IgnoreCarrier=" a bit too generic? if i undestanding this correctly, then this is a subset of ConfigureWithoutCarrier=, right? i.e. it is purely about tweaking behaviour when the carrier is lost, i.e. the edge from "carrier" to "no carrier", but not the edge from "no carrier" to "carrier", right?

maybe call this "MaintainWithoutCarrier=" or "RemainWithoutCarrier" or "KeepWithoutCarrier=" or so?